backend/dep/wire.go

```diff
@@ -36,14 +37,12 @@ import (
 	"github.com/short-d/short/dep/provider"
 )
 
-// TODO(issue#640): replace with value from env variable.
-const oneDay = 24 * time.Hour
-const oneWeek = 7 * oneDay
+var tokenDuration = getTokenDuration()
 
 var authSet = wire.NewSet(
 	provider.NewJwtGo,
 
-	wire.Value(provider.TokenValidDuration(oneWeek)),
+	wire.Value(provider.TokenValidDuration(tokenDuration)),
```

Comment:
We should stick the one way how to pass configurable values. Please take a look at how we do it `https://github.com/short-d/short/blob/master/backend/main.go#L54`